tests/bearlib/abstractions/LinterTest.py

```diff
@@ -38,15 +40,18 @@ class EmptyTestLinter:
 
     class RootDirTestLinter:
 
+        _root_dir = '/c' if platform.system() == 'Windows' else '/'
```

Comment:
why are those protected? Why not simply `ROOT_DIR` so they're like class global constants?